File: python/fizzbuzz/fizzbuzz_test.py
import unittest
from fizzbuzz import FizzBuzz

class TestFizzBuzz(unittest.TestCase):
    def setUp(self):
        self.obj = FizzBuzz()

    def test_1をいれたら1を返す(self):
        self.assertEqual(self.obj.run(1), "1")

    def test_7をいれたら7を返す(self):
        self.assertEqual(self.obj.run(7), "7")

    def test_3をいれたらFizzを返す(self):
        None#

    def test_9をいれたらFizzを返す(self):
        None#

    def test_5をいれたらBuzzを返す(self):
        None#

    def test_20をいれたらBuzzを返す(self):
        None#

    def test_15をいれたらFizzBuzzを返す(self):
        None#

    def test_45をいれたらFizzBuzzを返す(self):
        None#

File: golang/fizzbuzz/fizzbuzz_test.go
package fizzbuzz

import "testing"

func Test_1をいれたら1が返る(t *testing.T) {
	if fizzbuzz(1) != "1" {
		t.Error(`miss`)
	}
}

func Test_2をいれたら2が返る(t *testing.T) {
	if fizzbuzz(2) != "2" {
		t.Error(`miss`)
	}
}

func Test_3をいれたらFizzが返る(t *testing.T) {
	if fizzbuzz(3) != "Fizz" {
		t.Error(`miss`)
	}
}

func Test_6をいれたら6が返る(t *testing.T) {
	if fizzbuzz(6) != "Fizz" {
		t.Error(`miss`)
	}
}

func Test_5をいれたらBuzzが返る(t *testing.T) {
	if fizzbuzz(5) != "Buzz" {
		t.Error(`miss`)
	}
}

func Test_10をいれたらBuzzが返る(t *testing.T) {
	if fizzbuzz(10) != "Buzz" {
		t.Error(`miss`)
	}
}

func Test_15をいれたらFizzBuzzが返る(t *testing.T) {
	if fizzbuzz(15) != "FizzBuzz" {
		t.Error(`miss`)
	}
}

func Test_30をいれたらFizzBuzzが返る(t *testing.T) {
	if fizzbuzz(30) != "FizzBuzz" {
		t.Error(`miss`)
	}
}
